comicboxd/app/database/funcs.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/jmoiron/sqlx"
 )
 
-var debug = false
+var debug = true
```

Comment:
Should this change be committed to master?

Reply:
nope, I just removed it